terraform/modules/bosh_vpc/variables.tf

```diff
@@ -41,7 +41,7 @@ variable "nat_gateway_instance_type" {
 }
 
 variable "nat_gateway_ami" {
-  default = "ami-e8ab1489"
+  default = "ami-6ae2660b"
```

Comment:
Would it make sense to get the AMI ID programmatically instead of hard-coding, possibly using https://www.terraform.io/docs/providers/aws/d/ami.html? If not, maybe we can add a comment explaining what this refers to--or we can just wait for govcloud to offer managed nat gateways :trollface: